extensions-core/lookups-cached-single/src/main/java/io/druid/server/lookup/LookupExtractionModule.java

```diff
@@ -40,8 +41,7 @@
           @Override
           public void setupModule(SetupContext context)
           {
-            context.registerSubtypes(LoadingLookupFactory.class);
-            context.registerSubtypes(PollingLookupFactory.class);
+            context.setMixInAnnotations(LookupExtractorFactory.class, LookupExtractorFactoryMixIn.class);
```

Comment:
This is not commonly used in Druid, can the subtype registration be kept as per prior? If you want to keep names and class registration together you can use `com.fasterxml.jackson.databind.jsontype.NamedType`.